ablog/post.py

```diff
@@ -666,13 +666,17 @@ def generate_atom_feeds(app):
     if not url:
         return
 
+    base_url = url
+    if len(blog.blog_languages) > 1:
```

Comment:
Does `blog.blog_languages` return a list of languages?

Reply:
`blog.blog_languages` returns a dict of languages: https://ablog.readthedocs.io/en/latest/manual/ablog-configuration-options/#confval-blog_languages.

Reply:
Thanks.